docs/4_2_co-simulation_api.adoc

```diff
@@ -387,7 +389,7 @@ The state machine in <<figure-co-simulation-state-machine>> defines the supporte
 [#figure-co-simulation-state-machine]
 image::images/state-machine-co-simulation.svg[width=80%, align="center"]
 
-In this Co-Simulation interface the following functions must not be called: <<fmi3ActivateModelPartition>>, <<fmi3NewDiscreteStates>>, <<fmi3EnterEventMode>>, <<fmi3SetClock>>, <<fmi3GetClock>>, <<fmi3SetIntervalFraction>>, <<fmi3GetIntervalFraction>>, <<fmi3SetIntervalDecimal>>, <<fmi3GetIntervalDecimal>> including all functions that are specific to Model Exchange.
+In this Co-Simulation interface the following functions must not be called: <<fmi3SetTime>>, <<fmi3ActivateModelPartition>>, <<fmi3NewDiscreteStates>>, <<fmi3SetContinuousStates>>, <<fmi3EnterEventMode>>, <<fmi3SetClock>>, <<fmi3GetClock>>, <<fmi3SetIntervalFraction>>, <<fmi3GetIntervalFraction>>, <<fmi3SetIntervalDecimal>>, <<fmi3GetIntervalDecimal>> including all functions that are specific to Model Exchange.
```

Comment:
fmi3SetClock and fmi3GetClock are allowed in Co-Simulation

Reply:
Agreed, fixed.